Misc/NEWS.d/next/Documentation/2021-07-13-18-51-03.bpo-44628.lQlW93.rst

```diff
@@ -0,0 +1 @@
+Removed broken link for issue #445902 in the docstring of the runtime_library_dir_option() method of the UnixCCompiler class.
```

Comment:
Such tiny changes do not require a NEWS entry: https://devguide.python.org/triaging/#github-labels-for-prs